WTWSMS/common/traits/VIET_Immersion_Other.txt

```diff
@@ -86,6 +72,7 @@ israel_restorer = { # The Restorer of Israel
 	random = no
 	
 	same_religion_opinion = 10
+	jewish_opinion = 30
```

Comment:
A bosot that low is unrealistic: they would be a hero to all Jews, like the Prophet Muhammad's positive thirty boost gave.